scripts/prepare.js

```diff
@@ -27,7 +28,7 @@ const CLI_HUSKY = 'npx husky install';
 
 // this will create .husky/_/husky.sh if it does not yet exist.
 if (!existsSync(FILE_HUSKY)) {
-  global.console.log(CLI_HUSKY);
+  logger.info(CLI_HUSKY);
```

Comment:
Do we want to make this change? Surely we want the Winston logger to be used for Git Proxy related services only not necessarily development tooling?